### DISCUSSION TITLE
Low-pass filter in `PlayerLoader` should use delay instead of duration

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#19285] [Ideas] [answered]
I think clearing the low-pass filter for 300ms sounds jarring. I propose a delay is used instead:
```diff
-            lowPassFilter.CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF, CONTENT_OUT_DURATION);
+            lowPassFilter.Delay(CONTENT_OUT_DURATION).Then().CutoffTo(AudioFilter.MAX_LOWPASS_CUTOFF);
```

https://user-images.githubusercontent.com/11320483/180285690-a03f1ecb-c2d1-4122-9c24-48f11c489df3.mp4

https://user-images.githubusercontent.com/11320483/180285530-a8c0599b-c803-4066-9d30-847999871836.mp4

**Comment by peppy** [comment, marked as answer]
Thanks for the feedback. The existing effect is intentional.

**Comment by frenzibyte** [comment]
It feels off with the proposed change, since the low-pass filter is supposed to come when the player loader is present, so once it finishes loading, the low-pass filter should fade away before the track fades out.

**Reply by nagi-desuuu** [reply]
I should butt in that this proposed change may also affect beatmaps that have seizure warnings.

**Reply by LittleEndu** [reply]
To me the low pass filter does not signal the loading itself, instead it signals a pause before the beatmap. So it feels weird that the filter goes away before I get to play the beatmap I waited for.
